models/observationSubmissions.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   name: 'observationSubmissions',
   schema: {
     entityId: {
-      type: 'ObjectId',
+      type: String,
```

Comment:
@borkarsaish65 why there is a change in type? entityId will be mongoId right?

Reply:
the db i was using was test diksha db so in that entityId was string. 
I have changed it now.